src/main/java/org/project36/qualopt/service/github/GitHubAPIRequest.java

```diff
@@ -0,0 +1,110 @@
+package org.project36.qualopt.service.github;
+
+import org.project36.qualopt.service.github.GitHubUserType;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * A GitHub API Request that contains the information entered from the front end GitHub API
+ * User Search Form. The Springboot framework automatically marshalls this object when the
+ * GitHubResource receives a HTTP Request
+ */
+public class GitHubAPIRequest implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private int muleId;
+
+    private GitHubUserType userType; // convert to Enum later
```

Comment:
Is this comment redundant now? It looks like you have changed it to an enum

Reply:
Ah nice find - will remove that comment now!